main.go

```diff
@@ -1,11 +1,29 @@
 package main
 
-import "fmt"
-
+import (
+	"fmt"
+	"strconv"
+)
 
 func main() {
+
+	var status string = "IN PROGRESS"
```

Comment:
```suggestion
	var status string = "In Progress"
```
Next to you can usee strrings.ToUpper() method to uppercase all characters. This will give ability to use strings package